index.js

```diff
@@ -137,7 +137,7 @@ AudioBuffer.prototype.sampleRate = AudioBuffer.context.sampleRate || 44100;
  */
 AudioBuffer.prototype.getChannelData = function (channel) {
 	//FIXME: ponder on this, whether we really need that rigorous check, it may affect performance
-	if (channel >= this.numberOfChannels || channel < 0 || channel == null) throw Error('Cannot getChannelData: channel number (' + channel + ') exceeds number of channels (' + this.numberOfChannels + ')');
+	if (channel > this.numberOfChannels || channel < 0 || channel == null) throw Error('Cannot getChannelData: channel number (' + channel + ') exceeds number of channels (' + this.numberOfChannels + ')');
```

Comment:
We suppose that `0` goes for 1st channel, and `1` for 2nd channel, therefore `numberOfChannels` is always 1 more than maximum channel number. Just like `array.length`.